transaction/core/src/mint/validation/common.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Common validation code shared between different mint transaction types.
+
+use crate::{
+    mint::{
+        constants::{NONCE_MAX_LENGTH, NONCE_MIN_LENGTH},
+        validation::error::Error,
+    },
+    validation::{
+        validate_tombstone as transaction_validate_tombstone, TransactionValidationError,
+    },
+    BlockVersion, TokenId,
+};
+
+/// A wrapper around crate::validation::validate_tombstone that maps to our
+/// error type.
+///
+/// # Arguments
+/// * `current_block_index` - The index of the block currently being built.
+/// * `tombstone_block_index` - The block index at which this transaction is no
+///   longer considered valid.
+pub fn validate_tombstone(
+    current_block_index: u64,
+    tombstone_block_index: u64,
+) -> Result<(), Error> {
+    transaction_validate_tombstone(current_block_index, tombstone_block_index).map_err(|err| {
+        match err {
+            TransactionValidationError::TombstoneBlockExceeded => Error::TombstoneBlockExceeded,
+            TransactionValidationError::TombstoneBlockTooFar => Error::TombstoneBlockTooFar,
+            _ => Error::Unknown, /* This should never happen since validate_tombstone only
```

Comment:
Consider logging a warning here

Reply:
There's currently no logger being plumbed into here, and the crate does not depend on the logger feature, so I would prefer to skip that.